Resources/Prototypes/Species/slime.yml

```
@@ -36,19 +36,19 @@
       points: 1
       required: false
     Chest:
-      points: 1
+      points: 4 # imp
       required: false
     Legs:
-      points: 4
+      points: 8 # imp
       required: false
     Arms:
-      points: 4
+      points: 48 # imp
       required: false
 
 - type: humanoidBaseSprite
   id: MobSlimeMarkingFollowSkin
   markingsMatchSkin: true
-  layerAlpha: 0.5
+  layerAlpha: 0.75 # imp
```

```suggestion
  layerAlpha: 0.75 # imp 0.5<0.75
```
Note original value for future maintenance work.